btr-web/btr-common-components/interfaces/btrAddress.ts

```diff
@@ -0,0 +1,20 @@
+export type BtrCountry = {
```

Comment:
We've been using 'interface' instead of 'type' everywhere else. Doesn't seem to be much difference as both give you the same functionality, but I think we should stay consistent. I also started naming these files like `btr-address-i.ts` with each interface like `BtrCountryI`. I do it similarly for enums (i.e. `account-type-e.ts` / `AccountTypeE`). It makes it a little nicer when we have a lot of these files with folders expanded or when you have multiple interfaces / enums imported with similar names

Reply:
well, there are subtle differences, but the one I have issues with is that interfaces can be merged. Having them declared twice, you wont get error, but the interfaces will merge and create one.  

But if we all accept this as possibility I can change to interfaces, cause they are supposed to be faster for type checking :) .

Reply:
<img width="803" alt="Screen Shot 2023-10-26 at 11 22 10 AM" src="https://github.com/bcgov/business-transparency-registry/assets/16654897/df0eb023-60d3-42b3-ac4d-8b803d83ea36">